paasta_tools/cli/utils.py

```diff
@@ -742,3 +744,34 @@ def pick_slave_from_status(status, host=None):
     else:
         slaves = status.marathon.slaves
         return slaves[0]
+
+
+def get_task_from_instance(cluster, service, instance, slave_hostname=None, task_id=None):
```

Comment:
It's not clear to me why we're not reusing the logic that ``paasta status`` uses to get task info? This seems like a really limited subset of the things you'd get if you just used the mesos api?